fe/fe-core/src/main/java/org/apache/doris/analysis/CreateUserStmt.java

```diff
@@ -153,8 +153,8 @@ public void analyze(Analyzer analyzer) throws UserException {
 
         userIdent.analyze();
 
-        if (userIdent.isRootUser()) {
-            ErrorReport.reportAnalysisException(ErrorCode.ERR_COMMON_ERROR, "Can not create root user");
+        if (userIdent.isSystemUser()) {
```

Comment:
i think we should not prohibit create admin since admin maybe dropped in privous version